full-service/src/db/txo.rs

```diff
@@ -950,6 +962,46 @@ impl TxoModel for Txo {
         Ok(())
     }
 
+    fn update_is_synced_to_t3(&self, is_synced: bool, conn: Conn) -> Result<(), WalletDbError> {
+        use crate::db::schema::txos;
+
+        diesel::update(self)
+            .set(txos::is_synced_to_t3.eq(is_synced))
+            .execute(conn)?;
+
+        Ok(())
+    }
+
+    fn get_txos_that_need_to_be_synced_to_t3(
+        limit: Option<usize>,
+        conn: Conn,
+    ) -> Result<Vec<Txo>, WalletDbError> {
+        use crate::db::schema::txos;
+
+        let memo_types_predicate = txos::memo_type
+            .eq(Some(two_bytes_to_i32(
```

Comment:
Why does this `eq` requite `Some` but the others do not?

Reply:
That may have been from my autocomplete, I don't think the Some is required